src/uct/api/v2/uct_v2.h

```diff
@@ -198,6 +209,49 @@ typedef struct uct_md_mem_dereg_params {
 } uct_md_mem_dereg_params_t;
 
 
+/**
+ * @ingroup UCT_RESOURCE
+ * @brief Operation parameters passed to @ref uct_iface_is_reachable_v2.
+ */
+typedef struct uct_iface_is_reachable_params {
+    /**
+     * Mask of valid fields in this structure and operation flags, using
+     * bits from @ref uct_iface_is_reachable_v2. Fields not specified
+     * in this mask will be ignored. Provides ABI compatibility with respect
+     * to adding new fields.
+     */
+    uint64_t                     field_mask;
+
+    /**
+     * Device address to check reachability to. It is not set
+     * if iface_attr.dev_addr_len == 0, and must be set otherwise.
+     */
+    const uct_device_addr_t      *dev_addr;
+
+    /**
+     * Interface address to check reachability to. It is not set if
+     * iface_attr.iface_addr_len == 0, and must be set otherwise.
+     */
+    const uct_iface_addr_t       *iface_addr;
+
+    /**
+     * User-provided pointer to a string buffer.
+     * This value is optional.
```

Comment:
i think we need to remove `This value is optional` for all fields in this structure

Reply:
fixed